crafting-dead-core/src/main/java/com/craftingdead/core/client/gui/screen/inventory/GenericContainerScreen.java

```diff
@@ -41,6 +46,19 @@ public GenericContainerScreen(GenericMenu menu, Inventory playerInventory,
     this.imageHeight = 114 + this.menu.getRows() * AbstractMenu.SLOT_SIZE;
   }
 
+  @Override
+  public void init() {
+    super.init();
+    this.returnButton = CompositeButton.button(this.leftPos + 157, this.topPos - 1, 12, 16,
+            GENERIC_CONTAINER_TEXTURE)
+        .setAtlasPos(244, 0)
+        .setHoverAtlasPos(231, 0)
+        .setInactiveAtlasPos(8, 129)
+        .setAction((button) -> NetworkChannel.PLAY.getSimpleChannel()

```

Comment:
I dded it as a protected field that can be overwritten by children classes, trying to pass it as constructor would break the usage of `GenericContainer::new` (even with a constructor having the same old paramters)